.gitignore

```diff
@@ -103,7 +103,7 @@ wp-tests-config.php
 !/src/js/_enqueues/vendor
 
 # Files for local environment config
-/docker-compose.override.yml
```

Comment:
The new file should be added without removing the old one.

If a contributor has a `docker-compose.override.yml` file in there checkout and updates, then that file would be subject to version control.

You could switch the order of these and add an inline comment at the end of the old format. Something like `# The compose.override.yml is the new preferred Compose file name.`